share/templates/08-Orchestral/03-String_Orchestra.mscx

```diff
@@ -24,7 +22,6 @@
     <metaTag name="movementNumber"></metaTag>
     <metaTag name="movementTitle"></metaTag>
     <metaTag name="poet"></metaTag>
-    <metaTag name="source">http://musescore.com/score/820831</metaTag>
```

Comment:
good to see that gone

Reply:
Well I deleted that using notepad, otherwise it'll still be there ;-)

Reply:
it indeed just doesn't belong there, but was harmless I believe

Reply:
Well it maybe could do some harm if you make a score based on that template and click "Save Online"... But you still don't have write access to that url so not sure what'll happen...